apps/voting/contracts/Voting.sol

```diff
@@ -226,7 +248,7 @@ contract Voting is IForwarder, AragonApp {
     }
 
     /**
-    * @notice Tells whether `_sender` can participate in the vote #`_voteId` or not
+    * @notice Tells whether `_sender` can vote in the vote #`_voteId` by voting 'yes' or 'no'
```

Comment:
@notice Tells whether `_sender` can participate in the main phase of the vote #`_voteId`